tools/initializer/templates/model/gauges.yml.ftl

```diff
@@ -0,0 +1,37 @@
+#
+# Gauge Type and Gauge Instance Specifications
+#  - time periods generally in milliseconds
+gauge-types:
+  [=gaugeTypeName]:
+    commands:
+      load :
+    setupParams:
+      targetIP:
```

Comment:
Is it a good idea to leave these fields empty? Should fill in a variable if possible.